dev-client/src/components/common/EmbadgedIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import {Badge, Pressable} from 'native-base';
+import {IconButton} from 'terraso-mobile-client/components/common/Icons';
+
+type Props = {
+  _badge?: {} & Omit<React.ComponentProps<typeof Badge>, 'variant'>;
+  _iconButton?: {} & Omit<
+    React.ComponentProps<typeof IconButton>,
+    'onPress' | 'name' | 'accessibilityLabel'
+  >;
+  badgeNum: number;
+  iconName: string;
+  accessibilityLabel?: string;
+} & React.ComponentProps<typeof Pressable>;
+
+const EmbadgedIcon = ({
```

Comment:
question: is Embadged a common React Native or NativeBase term? If not, please call this `BadgedIcon`.

Reply:
@david-code Please explain more about the name.

Reply:
Should be changed now.